Templates/EventHubTrigger-CSharp/function.json

```diff
@@ -6,7 +6,8 @@
             "name": "myEventHubMessage",
             "direction": "in",
             "path": "samples-workitems",
-            "connection": ""
+            "connection": "",
+            "consumerGroup": "$Default"
```

Comment:
Typically non mandatory fields are not added to templates, if the customer needs that explicitly they can add the value from integration tab, which then will just add the setting to function.json